proposals/0439-trailing-comma-lists.md

```diff
@@ -280,9 +280,9 @@ protocol Foo {
 
 ```swift
 protocol Foo {
-  associatedtype T:
-      P1,
-      P2, ❌ Expected type
+  func f<T1, T2>(a: T1, b: T2) where
```

Comment:
While this was probably intended, instead of adding new text, we could delete the unintended repeated example altogether.

Reply:
@xwu Done! I removed the repeated example and updated the PR title. Thanks.